lisa/wlgen/rta.py

```diff
@@ -46,6 +46,8 @@ class RTA(Workload):
 
     sched_policies = ['OTHER', 'FIFO', 'RR', 'DEADLINE']
 
+    ALLOWED_TASK_NAME_REGEX = r'^[a-zA-Z0-9_]+$'
```

Comment:
Why not dashes? Is it for easier regexp'ing due to e.g. forking? Just curious.

Reply:
Dashes followed by numbers can introduce ambiguity since rt-app can use one or more `-XX` prefixes. If rt-app decides to add further suffixes, it will likely be based on even more dashes, so I'd rather avoid them altogether at our level